pkg/test/detector/curltest.go

```diff
@@ -31,6 +32,14 @@ func (d CurlTestAdapter) Is(options apiClient.UpsertTestOptions) (name string, o
 	return
 }
 
+// IsWithPath detects based on upsert test options what kind of test it is
+func (d CurlTestAdapter) IsWithPath(path string, options apiClient.UpsertTestOptions) (name string, ok bool) {
```

Comment:
should we move it to executor folder and create detector, validator packages? we can register them at app start

Reply:
Yes, I think it is a good idea to move them to executors.